config.yaml.template

```diff
@@ -35,7 +35,5 @@ model:
         - 0.225
         - 0.225
 
```

Comment:
It would be good if we can document what each of the fields would configure here in the config file. This way every time one adds a new field they add a comment about what it is

Reply:
I will add it in a seperate branch